app/src/main/java/ch/sdp/vibester/activity/GameEndingActivity.kt

```diff
@@ -140,4 +142,32 @@ class GameEndingActivity : AppCompatActivity() {
             }
         }
     }
+
+    override fun onItemClick(position: Int) {
```

Comment:
Maybe use some empty lines to separate logic and improve readability

Reply:
Will be on the next commit!